msteams-platform/concepts/tabs/tabs-overview.md

```diff
@@ -2,10 +2,12 @@
 title: Add tabs to Microsoft Teams apps
 description: Describes how to get started developing tabs in Microsoft Teams
 keywords: teams tabs development
-ms.date: 11/12/2018
 ---
 # Add tabs to Microsoft Teams apps
 
+> [!Important]
+> Full support for static tabs on mobile clients is currently in [developer preview](~/resources/dev-preview/developer-preview-intro.md), and will be released soon. To prepare for this change you should follow the [guidance for personal apps on mobile](~/resources/design/framework/tabs-mobile.md) when creating your tabs. When full support for tabs is release, your `contentUrl` **will be loaded in the mobile Teams client**. Your users will still have the option of opening your tab in a separate browser via your `websiteUrl`, however your `contentUrl` will be loaded first.
```

Comment:
We should remove this statement from static tab info here:
Your users will still have the option of opening your tab in a separate browser via your `websiteUrl`, however your `contentUrl` will be loaded first.